docs/examples.md

```diff
@@ -19,7 +41,54 @@ const nextConfig = {
 module.exports = nextConfig;
 ```
 
-## [Edge API Routes](https://nextjs.org/docs/api-routes/edge-api-routes)
+</details>
+
+### App Directory
+
```

Comment:
Because it is no longer needed now that the app directory is stable. If someone wants to use an old beta version of the app directory, they would have to use the setting, but it wouldn't exactly be a wise choice to use an old beta version when there it has been changed to stable.